app/controllers/organisation_member/shares_controller.rb

```diff
@@ -5,18 +5,20 @@ class OrganisationMember::SharesController < OrganisationMember::BaseController
 
   # GET /shares
   def index
-    @shares = @organisation.shares
+    @shares = policy_scope(@organisation.shares)
   end
 
   # GET /shares/1
   def show
     @official_birth_record = @share.access(accessed_by: current_account)
+    authorize @official_birth_record
```

Comment:
i couldn't find a solution here-- we log the access before we check they're allowed. But we do check `@share` on `set_share` so it's fine.

Reply:
This is changed around in #175 anyway so I'll have to revisit it there